### DISCUSSION TITLE
[NextJs] Global CSS cannot be imported from within node_modules.

### DISCUSSION THREAD

**Discussion by FFF-F1** [antvis/Graphin#398] [Q&A]
Hi, thanks for your great work of this library. I'm trying to use it in my NextJs app but it throws an error. Seems like NextJs doesn't support importing CSS from node_modules so could you separate style from the index.js? Thanks in advance.

BTW, the full error information is [here](https://github.com/vercel/next.js/blob/master/errors/css-npm.md).

**Comment by janwirth** [comment]
I have the same problem. Did you solve it?
